clients/cli/src/main/java/org/apache/gravitino/cli/commands/Command.java

```diff
@@ -69,4 +77,17 @@ protected GravitinoAdminClient buildAdminClient() {
       return GravitinoAdminClient.builder(url).build();
     }
   }
+
+  /**
+   * Outputs the entity to the console.
+   *
+   * @param entity The entity to output.
+   */
+  protected <T> void output(T entity) {
+    if (outputFormat != null && outputFormat.equals("table")) {
+      TableFormat.output(entity);
+    } else {
```

Comment:
Should also check if the output is "plain"? May not be best to hardcode "table"?

Reply:
If the user does not specify the output , the default is PlainFormat. Static variables are used to store "table" and "plain"; this has been fixed in commit: https://github.com/apache/gravitino/pull/5606/commits/675bf45496db13a65dd1d5238b4dd20eb44f08de

Reply:
Understand. This has been fixed in commit: https://github.com/apache/gravitino/pull/5606/commits/21dabc397c4299ace464fe28148c8a97fdc807e8